dist/css/violegacy.css

```diff
@@ -1264,18 +1269,204 @@ button:hover{
 .add-referral-template{
     margin:3vh;
     flex:0 1 25%;
-    border: 0.5vh dashed grey;
+    border: 0.5vh dashed rgb(165, 164, 164);
+    color:grey;
     display:flex;
     flex-direction: column;
     justify-content: center;
     align-items: center;
 }
 
 .add-referral-template div{
-    color: grey;
+    color: rgb(165, 164, 164);
     font-size: 600%;
 }
 
 .alumni-opp-container{
     margin-top: 5vh;
 }
+
+
+
+/*Tests for senior-myOpp-detail*/
```

Comment:
Can we make sure the code is as clean as possible?

Reply:
Done